sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ControllingPermissionsTest.cs

```diff
@@ -47,29 +47,36 @@ public class ControllingPermissionsTest
         [TestCategory("Rule")]
         public void ControllingPermissions_CS() =>
             Verifier.VerifyAnalyzer(@"TestCases\ControllingPermissions.cs",
-                new CSharp.ControllingPermissions(AnalyzerConfiguration.AlwaysEnabled),
-                additionalReferences: AdditionalReferences);
+                                    new CSharp.ControllingPermissions(AnalyzerConfiguration.AlwaysEnabled),
+                                    additionalReferences: AdditionalReferences);
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void ControllingPermissions_CSharp9() =>
```

Comment:
Do we need this test? This whole file is wrapped in 
```
#if NETFRAMEWORK // IdentityModel is not available on .Net Core
```
so there's no way for the production code to be run under .NET 5 / C# 9.

Reply:
No, nice catch.